src/components/organisms/Menu/style.scss

```diff
@@ -0,0 +1,84 @@
+.header__menu {
+    position: fixed;
+    top: 60px;
+    left: -100%;
+    width: 100%;
+    height: 100%;
+    background-color: var(--main-bg-color);
+    transition: left 0.5s;
+    overflow: auto;
+    z-index: 20;
+
+    &.active {
+        left: 0;
+    }
+
+    .header__list {
+        padding: 0 10px 10px;
+        text-transform: uppercase;
+        font-size: 12px;
+        letter-spacing: 1px;
+        font-family: var(--main-text-font);
+        cursor: pointer;
+        justify-self: end;
+    
+        li {
+            list-style: none;
+            margin: 0 0 10px;
+            border-top: 2px solid var(--main-border-color);  
+            padding: 7px 10px 0;
+            line-height: 35px;
+    
+            &:first-child {
+                border-top: 0;    
+            }
+
+            .header__link, .header__sublink {
+                text-decoration: none;
+                color: var(--main-color);
+            
+                &:hover {
+                    color: var(--brown-color);
+                }
+            }
+        }
+
+        .header__submenu {
+            position: fixed;
+            top: 60px;
+            left: 100%;
+            width: 100%;
+            padding: 0 10px 0 5px;
+            background-color: var(--main-bg-color);
+            transition: left 0.5s;
+            overflow: auto;
+            z-index: 20;
+        
+            &.active {
```

Comment:
why not &--active , to this header__submenu--active  would be more valid BEM class naming

Reply:
Changed to &--active